src/LiveComponent/src/ComponentWithFormTrait.php

```diff
@@ -137,16 +137,16 @@ public function getFormName(): string
     private function resetForm(): void
     {
         // prevent the system from trying to submit this reset form
+        $this->formValues = $this->extractFormValues($this->getFormView());
         $this->shouldAutoSubmitForm = false;
         $this->form = null;
         $this->formView = null;
-        $this->formValues = $this->extractFormValues($this->getFormView());
     }
 
     private function submitForm(bool $validateAll = true): void
     {
         if (null !== $this->formView) {
-            throw new \LogicException('The submitForm() method is being called, but the FormView has already been built. Are you calling $this->getForm() - which creates the FormView - before submitting the form?');
+            throw new \LogicException('The submitForm() method is being called, but the FormView has already been built. Are you calling $this->getForm() - which creates the FormView - before submitting the form? Or did you forget to call $this->resetForm() to reset the form state to support bulk requests?');
```

Comment:
Could you create a small reproducer with bulk requests ? What i'm not sure to get is how bulk requests would interfer .. the component beeing non shared services, they should not receive multiple request, no ?